internal/operator-controller/rukpak/bundle/config.go

```diff
@@ -6,119 +6,498 @@ import (
 	"fmt"
 	"strings"
 
+	"github.com/santhosh-tekuri/jsonschema/v6"
```

Comment:
maybe we should move config.go, config_test.go, and error_formatting_test.go out of this package and into a top level `bundle` package? wdyt? Maybe just leave the registry+v1 bundle building code here or move it to registryv1.go?